app/src/main/res/values-land/dimens.xml

```diff
@@ -6,4 +6,5 @@
   <dimen name="home_margin_min">36dp</dimen>
   <dimen name="recently_played_margin_max">72dp</dimen>
   <dimen name="recently_played_margin_min">36dp</dimen>
+  <dimen name="profile_view_already_added_margin">20dp</dimen>
```

Comment:
Given that this is the same for portrait and landscape, we don't need to redefine it here.

Reply:
@ARJUPTA This can be removed.

Reply:
Resolving this on behalf of @BenHenning